torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -2329,6 +2329,13 @@ void InitXlaModuleBindings(py::module m) {
           }
           return result;
         });
+  m.def("_xla_mark_bounded_dynamic",
+        [](const at::Tensor& input, const std::vector<uint32_t>& dims,
+           const std::vector<uint32_t>& bounds) {
+          TORCH_LAZY_COUNTER("XlaMarkBoundedDynamic", 1);
+          XLATensorPtr xtensor = bridge::GetXlaTensor(input);
+          xtensor->MarkBoundedDynamicDimension(dims, bounds);
```

Comment:
Is there a checking to ensure that len(dims) is equal to len(bounds)?

Reply:
I check this in torchacc, also add simple check here.